src/sql/src/func.rs

```diff
@@ -422,19 +424,19 @@ pub struct FuncImpl<R> {
 #[derive(Debug)]
 pub struct FuncImplCatalogDetails {
     pub oid: u32,
-    pub arg_oids: Vec<u32>,
-    pub variadic_oid: Option<u32>,
-    pub return_oid: Option<u32>,
+    pub arg_oids: Vec<GlobalId>,
+    pub variadic_oid: Option<GlobalId>,
+    pub return_oid: Option<GlobalId>,
     pub return_is_set: bool,
 }
 
 impl<R: GetReturnType> FuncImpl<R> {
     fn details(&self) -> FuncImplCatalogDetails {
         FuncImplCatalogDetails {
             oid: self.oid,
-            arg_oids: self.params.arg_oids(),
-            variadic_oid: self.params.variadic_oid(),
-            return_oid: self.return_type.typ.as_ref().map(|t| t.oid()),
+            arg_oids: self.params.arg_ids(),
+            variadic_oid: self.params.variadic_id(),
+            return_oid: self.return_type.typ.as_ref().map(|t| t.global_id()),
```

Comment:
Should these all be names `X_ids` instead of `X_oids` or am I misunderstanding?

Reply:
Yeah, just a missed rename